.github/workflows/build-rapids.yml

```diff
@@ -90,11 +90,37 @@ jobs:
           #! /usr/bin/env bash
           set -eo pipefail
 
-          . ~/cccl/ci/rapids/post-create-command.sh;
-
           declare -a failures
           declare -A failures_map
 
+          _print_err_exit_msg() {
+            local code=$?
+            if test $code -ne 0; then
+              echo "::error:: Failures: ${failures[*]}"
+              echo -e "::group::️❗ \e[1;31mInstructions to Reproduce CI Failure Locally\e[0m"
+              echo "::error:: To replicate this failure locally, follow the steps below:"
+              echo "1. Clone the repository, and navigate to the correct branch and commit:"
+              echo "   git clone --branch $GITHUB_REF_NAME --single-branch https://github.com/$GITHUB_REPOSITORY.git && cd $(echo $GITHUB_REPOSITORY | cut -d'/' -f2) && git checkout $GITHUB_SHA"
+              echo ""
+              echo "2. Run the failed command inside the same Docker container used by this CI job:"
+              cat <<____EOF
+             RAPIDS_LIBS='${RAPIDS_LIBS}'$(for lib in cmake ${RAPIDS_LIBS}; do var=RAPIDS_${lib//-/_}_GIT_REPO; if test -v "$var" && test -n "${!var}"; then echo -n " $var='${!var}'"; fi; done) \\
+               .devcontainer/launch.sh -d -c ${{matrix.cuda}} -H rapids-conda -- ./ci/rapids/rapids-entrypoint.sh \\
+               /bin/bash -li -c 'uninstall-all -j -qqq && clean-all -j && build-all -j -v || exec /bin/bash -li'
+          ____EOF
+              echo ""
+              echo "For additional information, see:"
+              echo "   - DevContainer Documentation: https://github.com/NVIDIA/cccl/blob/main/.devcontainer/README.md"
+              echo "   - Continuous Integration (CI) Overview: https://github.com/NVIDIA/cccl/blob/main/ci-overview.md"
+            fi
+            exit $code
+          }
+
+          # Print failures and exit
+          trap '_print_err_exit_msg' EXIT;
```

Comment:
I moved this into an exit trap because the sccache failures seem to cause the script to exit immediately, instead of being handled by the `if` conditional and failing gracefully.